### PR TITLE
Feat/add connectkit

### PR DESCRIPTION
**Pull request by q20274982**
## Summary

- add connectkit support
image
## Related Links

**Asana**: https://app.asana.com/0/1201812548509877/1204141696149542/f

## Checklist

- [x] Pasted Asana link.
- [x] Tagged labels.

### CODE REVIEW

File: adapters/connectkit-connector/src/bloctoConnector.ts [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    super({ chains, options });
+    this.#onAccountsChangedBind = this.onAccountsChanged.bind(this);
+    this.#onChainChangedBind = this.onChainChanged.bind(this);
+    this.#onDisconnectBind = this.onDisconnect.bind(this);
+  }
+
+  getProvider({ chainId }: { chainId?: number } = {}): Promise<BloctoProvider> {
+    if (!this.#provider) {
+      const { appId, ...rests } = this.options;
+      const _chainId = chainId ?? rests?.chainId ?? this.chains[0]?.id;
```
Comment (sanyu1225) [inline adapters/connectkit-connector/src/bloctoConnector.ts:68]: image
show @deprecated message ?!

File: .changeset/three-parents-exist.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@blocto/connectkit-connector': patch
```
Comment (scottphc) [inline .changeset/three-parents-exist.md:2]: Should be `minor`?

File: adapters/connectkit-connector/src/bloctoConnector.ts [hunk 3]
```diff
@@ -0,0 +1,232 @@
+import {
```
Comment (sanyu1225) [inline adapters/connectkit-connector/src/bloctoConnector.ts:1]: should use `@blocto/wagmi-connector` and override some function ?

Comment (q20274982) [reply adapters/connectkit-connector/src/bloctoConnector.ts:1]: I anticipate solving this issue in the next enhancement version. Until then, I need more testing to ensure the compatibility of wagmi-connector.

**Review by sanyu1225 — DISMISSED** [review]

**Review by akira02 — APPROVED** [review]

**Pull request merged**